source/beamline/BioXAS/BioXASBeamline.cpp

```diff
@@ -230,6 +247,10 @@ AMAction3* BioXASBeamline::createScanInitializationAction(AMGenericStepScanConfi
 
 		if (fastShutterInitialization)
 			initializationAction->addSubAction(fastShutterInitialization);
+
+		// Initialize the scan axis controls.
+
+		initializationAction->addSubAction(AMBeamline::bl()->createScanInitializationAction(configuration));
```

Comment:
Since the initialize the scan axis controls is moved to `AMBeamline::createScanInitializationAction()` from the `AMGenericStepScanController::createInitializationActions()`, the `createScanInitializationAction()` implementation of each beamline has to call this to get that feature. If we keep this changes, we need to add the same line of codes in `IDEASBeamline::createScanInitializationAction()`.